src/hypothesis/strategies.py

```diff
@@ -1015,13 +1012,14 @@ def fractions(min_value=None, max_value=None, max_denominator=None):
     be None or a positive integer.
 
     """
+    check_valid_bound(min_value, 'min_value')
```

Comment:
I don't think that's an unreasonable thing to do actually. The standard library error messages are generally quite bad, and improving them at the boundary is something we could easily do.